docs/product/performance/queue-monitoring/index.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: "Queue Monitoring"
+sidebar_order: 70
+description: "Learn how to monitor your queues with Sentry for improved application performance and health. "
+---
+
+<Include name="feature-stage-alpha.mdx" />
+
+Message Queues make asynchronous service-to-service communication possible in distributed architectures. Queues are great for making work that sometimes fails, more resilient and are therefore a building block for distributed applications. Some examples of what queues can help with include handling webhooks from third-party APIs or handling periodic tasks (such as calculating metrics for your users daily).
+
+If you have performance monitoring enabled and your application interacts with message queue systems, you can configure Sentry to monitor their performance and health.
+
+Queue monitoring allows you to monitor both the performance and error rates of your queue consumers and producers, providing observability into your distributed system.
+
+The **Queues** page gives you a high-level overview so that you can see where messages are being written to. (You may see topic names or actual queue names, depending on the messaging system.) If you click on a transaction, you'll see the **Destination Summary** page, which provides metrics about specific endpoints within your applications that either write to, or read from the destination. You can also dig into individual endpoints within your application representing producers creating messages, and consumers reading messages. You'll see actual traces representing messages processed by your application.
+
+### Prerequisites and Limitations
+
+Queue monitoring currently supports [auto instrumentation](/platform-redirect/?next=%2Fperformance%2Finstrumentation%2Fautomatic-instrumentation) for the [Celery Distributed Task Queue](https://docs.celeryq.dev/en/stable/) in Python. Other messaging systems can be monitored using custom instrumentation.
+
+Instructions for custom instrumentation in various languages are linked to below:
+
+- [Python SDK](/platforms/python/performance/instrumentation/custom-instrumentation/)
+- JavaScript SDKs - [NEW PAGE JavaScript → Setup Performance → Queue]
```

Comment:
Adding note here that we need to link this to the new SDK page for queues.

We should see if there's a similar one being added for Python we want to link to.

Reply:
@lizokm @bcoe I've merged in the javascript manual instrumentation to this PR and updated this link to point to the new page!